plugins/helphub-post-types/assets/css/admin.css

```diff
@@ -1,9 +1,612 @@
 .post-type.edit-php table.wp-list-table .column-image {
-	width: 120px;
-	text-align: center;
+  width: 120px;
+  text-align: center; 
 }
 
 .post-type.edit-php table.wp-list-table .column-image img {
 	width: 60px;
-	height: auto;
+	height: auto; 
 }
+
+
```

Comment:
What are your thoughts? - I only took what's needed for jQuery UI Datepicker. It's unlikely the CSS will ever change. And the file we will update is almost always only going to be the js file.